diagnostic.md

```diff
@@ -21,13 +21,18 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts/
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function () {
+  return $.ajax({
+    url: config.apiOrigin + '/donuts/',
+    method: 'GET',
+  });
+};
 ```
```

Comment:
This would work with the URL being built by config and apiOrigin files but we can't assume we have those. This is looking for you to heard code the URL as you did in your curl request. 

```js
let getDonuts = function () {
  return $.ajax({
    url: 'http://www.example.com/donuts',
    method: 'GET',
  });
};
```